### ISSUE TITLE
Docker Dir create when no docker user exists?

### ISSUE THREAD

**Issue by schoebelh** [ct-Open-Source/telerec-t-debian#1]
The main task tries to create the docker dir with "user = docker". This user does bot exist when initial installing an server - the docker install role runs after the system role.

**Comment by dm2002** [comment]
Same problem here, added a small pull request to create the role before.
https://github.com/ct-Open-Source/telerec-t-debian/pull/2

**Comment by stueken** [comment]
would be nice if somebody could just apply the PR, otherwise its a mess to handle possible updates with this repository as a git submodule.

**Comment by dm2002** [comment]
Yes, that would be great, but I'm afraid nobody cares about this repo

**Comment by stueken** [comment]
Thanks for merging. Can then be closed here.

**Comment by Markus-Be** [comment]
@schoebelh if the issue is solved for you? Can you please close it?
